website/docs/d/control_plane_ip_addresses.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: mongodbatlas_control_plane_ip_addresses"
+sidebar_current: "docs-mongodbatlas-datasource-control-plane-ip-addresses"
+description: |-
+    "Provides a data source that returns all control plane IP addresses"
+---
+
+# Data Source: mongodbatlas_control_plane_ip_addresses
+
+
+Provides a data source that returns all control plane IP addresses.
+
+## Example Usages
+```terraform
+data "mongodbatlas_control_plane_ip_addresses" "test" {
+}
+
+output "outbound-aws-ip-addresses" {
+  value = data.mongodbatlas_control_plane_ip_addresses.test.outbound.aws
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Read-Only
+
+- `inbound` (Attributes) List of inbound IP addresses to the Atlas control plane, categorized by cloud provider. If your application allows outbound HTTP requests only to specific IP addresses, you must allow access to the following IP addresses so that your API requests can reach the Atlas control plane. (see [below for nested schema](#nestedatt--inbound))
```

Comment:
added period at end + removed parenthetical as the last phrase is a complete sentence.
```suggestion
- `inbound` (Attributes) List of inbound IP addresses to the Atlas control plane, categorized by cloud provider. If your application allows outbound HTTP requests only to specific IP addresses, you must allow access to the following IP addresses so that your API requests can reach the Atlas control plane. See [below for nested schema](#nestedatt--inbound).
```

Reply:
Unfortunately this part of the text is autogenerated by an external tool. Can look into raising an issue if this is a concern.